src/generator/css/base.less

```diff
@@ -1025,3 +1573,425 @@ body.post-page {
+    padding-left: 1.25rem;
+    font-size: 28px;
+    font-family: var(--font-h1);
+    font-weight: 700;
+    border: unset;
+    text-transform: unset;
+    line-height: 1.25;
+
+    @media (min-width: 640px) {
+        padding-left: 0.75rem;
+        margin-left: 3rem;
+    }
+    
+    span {
+        @media (max-width: 600px) {
+            color: var(--color-primary);
+        }
+    }
+}
+
+.embla {
+    margin: auto;
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+    padding-left: 2rem;
+    --slide-height: 19rem;
+    --slide-spacing: 1rem;
+    --slide-size: 100%;
+  }
+
+  .embla__viewport {
+        overflow-x: hidden;
+  }
+
+  .embla__container {
+    display: flex;
+    align-items: center;
+    touch-action: pan-y pinch-zoom;
+    margin-left: calc(var(--slide-spacing) * -1);
+
+      .embla__slide:nth-child(odd) {
+        height: 485px;
+
+        p:first-of-type {
+            -webkit-line-clamp: 8; 
+        }
+      }
+  
+      .embla__slide:nth-child(even) {
+        height: 352px;
+
+        p:first-of-type {
+            -webkit-line-clamp: 4; 
+        }
+      }
+  
+      > .embla__slide:nth-child(1),
+      > .embla__slide:nth-child(4) {
+        background-color: var(--color-primary);
+      }
+  
+      > .embla__slide:nth-child(2),
+      > .embla__slide:nth-child(5) {
+        background-color: var(--color-secondary-offset);
+      }
+  
+      > .embla__slide:nth-child(3),
+      > .embla__slide:nth-child(6) {
+        background-color: var(--color-accent);
+      }  
+    }
+
+  .embla__slide {
+    transform: translate3d(0, 0, 0);
+    flex: 0 0 var(--slide-size);
+    max-width: 400px;
+    margin-left: 2.75rem;
+    padding-left: var(--slide-spacing); 
+
+    @media (max-width: 460px) {
+        max-width: 300px;  
+    }
+  }
+
+  .embla__viewport .embla__container .embla__slide .doc {
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+    padding-left: 2.75rem;
+    padding-right: 2.75rem;
+    font-size: 14px;
+    font-weight: 600;
+    display: flex;
+    flex-wrap: wrap;
+    user-select: none;
+    height: 100%;
+    
+
+    /* Max Text height is set with webkit */
+    p {
+        @apply text-base sm:text-2xl tracking-widest;
+        display: -webkit-box; 
+        -webkit-box-orient: vertical; 
+        overflow: hidden; 
+        text-overflow: ellipsis; 
+        clear: right;
+        font-weight: 400;
+        margin-bottom: auto;
+        text-overflow: ellipsis;
+      }
+
+      img {
+          @apply rounded-full inline mt-auto;
+          margin-top: auto;
+          width: auto;
+          height: auto;
+          max-width: none;
+          max-height: none;
+          width: 75px;
+          height: 75px;
+          flex: 0 0 75px;
+          object-fit: cover;
+          object-position: center;
+
+        + p {
+          flex: 1 1 50%;
+          @apply ml-4 mb-5 mt-auto;
+          display: inline;
+          margin-top: auto;
+        }
+      }
+  }
```

Comment:
if you imported these styles from embla it might be worth it to save them to a separate file over our own file. if not, please consider using more tailwind declarations here.